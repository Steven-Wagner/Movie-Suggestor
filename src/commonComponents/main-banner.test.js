import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import MainBanner from './main-banner';

describe('<MainBanner/>', () => {
    it('renders without crashing', () => {
    const div = document.createElement('div');

    ReactDOM.render(
        <Router>
            <MainBanner />
        </Router>,
        div);
    ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the UI as expected', () => {
        const tree = renderer
        .create(
            <Router>
                <MainBanner/>
            </Router>)
        .toJSON();
        expect(tree).toMatchSnapshot();  
    });
})