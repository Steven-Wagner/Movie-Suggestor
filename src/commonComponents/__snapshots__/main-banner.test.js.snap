// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainBanner/> renders the UI as expected 1`] = `
<header
  className="main-banner"
  role="banner"
>
  <h1>
    <span
      className="fas fa-film"
    />
    Movie Suggestor
  </h1>
  <h2
    className="subtitle"
  >
    Personalized Movie Suggestions
  </h2>
</header>
`;
